### PR TITLE
Notes for Develop with Flask

### PR DESCRIPTION
**Pull request by modelearth**

### CODE REVIEW

**Comment by modelearth** [comment]
@googlebot I signed it!

**Review by shifucun — COMMENTED** [review]
Thanks for the fix! I just submitted https://github.com/datacommonsorg/website/pull/805, can you sync this file?

File: developer_guide.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ git submodule foreach git pull origin master
 
 ## Prerequisites
 
+You can skip the prerequisites by using the Develop with Flask steps below to call the API from a local build.
```
Comment (shifucun) [inline developer_guide.md:21]: Can you move the 3 bullet point about GCP permission to a new section below like:

## (Optional) Get GCP permission

This is only needed when developing place search functionality.

- ...
- ...
- ...

**Comment by shifucun** [comment]
@modelearth do you still plan to work on this PR? If not I can close it. Thanks!

**Comment by modelearth** [comment]
You can close it. Thanks.

**Pull request closed without merging**